CheapShopWeb/Models/Product.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Web;
+
+namespace CheapShopWeb.Models
+{
+    [Table("novastar", Schema = "public")]
```

Comment:
ar negalim sudaryti table lista ir sujungti visus table?

Reply:
Ką turi tuo omenyje?

Reply:
tuomet gal nesuprantu ka daro [Table("novastar", Schema = "public")], gali paaiskint?

Reply:
Is duombazes pasirenka lentele novastar is schemos public, manyčiau reikės visus produktus sudeti i viena lentele ir gal susieti su kazkokiom kitom